pkg/cluster/sync.go

```diff
@@ -73,20 +73,6 @@ func (c *Cluster) Sync(newSpec *acidv1.Postgresql) error {
 		}
 	}
 
-	// create database objects unless we are running without pods or disabled that feature explicitly
```

Comment:
what caused  this change ? re-shuffling code by an editor ?

Reply:
Look at the logs from my comment in the other PDB PR: https://github.com/zalando/postgres-operator/pull/384#discussion_r287337006

Because of the error messages, first I thought that the sync code exited to early. That's why I moved PDB syncing before db syncing. But then I realized, I was looking at the logs of an UPDATE event, where PDB syncing was missing. I have introduced it there - also before the DB syncing. Therefore, I kept the change here to have the same syncing order. This way K8s resources are synced before DB stuff. Feels better to me this way.